content/docs/start/data-management.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Data Management Trail
+---
+
+As its name implies, DVC is used to control versions of data. It enables to keep
+track of multiple versions of your datasets.
```

Comment:
always better to include models ... in this case we might even include just "large files"?

Reply:
I think the trail name can be "Data and Model Management", BTW. Rename at this early stage?

Reply:
sounds good to me. Even though "model management" should be about metrics to some extent ...